src/app/pages/signing/signing.component.css

```
@@ -1,17 +1,68 @@
-.form-control:focus {
-  border-color: var(--accent);
-  box-shadow: none;
+#options > * {
+	margin: 0.5em 0.5em;
 }
 
-#options > * {
-  margin: 2em 0.5em;
+#title-div {
+	text-align: center;
+}
+
+#fontSelectorDropdown {
+	text-align: right;
+	width: 12em;
 }
 
 .format-icons {
-  width: 1.8em;
-  height: 1.8em;
+	width:1.8em;
+	height:1.8em;
+}
+
+.btn-format {
+	width: 2.4em;
+}
+
+#signing-header {
+	margin-left: 0.5em;
+	margin-right: 0.5em;
+  font-size: 1.6em;
+  text-align: center;
+}
+
+
+#signing-box {
+	margin: 1em;
+	padding: 1em;
+	width: 50em;
+	resize: none;
+	font: 16px Arial;
+	border-width: 0.05em;
+	border-radius: 0em;
```

```suggestion
	border-radius: 0;
```
Remove this elsewhere tooo